src/realizations/catchment/Bmi_Py_Formulation.cpp

```diff
@@ -43,11 +43,13 @@ void Bmi_Py_Formulation::protected_create_formulation(geojson::PropertyMap prope
     auto it_other_vars = properties.find(CFG_PROP_KEY_OTHER_INPUT_VARS);
     if (it_other_vars != properties.end()) {
         // TODO: consider adding error message if something in list other than input variables (output and/or unrecognized)
-        set_bmi_model(std::make_shared<Bmi_Py_Adapter>(Bmi_Py_Adapter(py_class, config_file, it_other_vars->second)));
+        int a = 1;
```

Comment:
@robertbartel Same issue here with the build errors.

ngen_pybind/ngen/extern/pybind11/include/pybind11/numpy.h:1160:37: error: static assertion failed: Attempt to use a non-POD or unimplemented POD type as a numpy dtype
 1160 |     static_assert(is_pod_struct<T>::value, "Attempt to use a non-POD or unimplemented POD type as a numpy dtype");
      |                                     ^~~~~

Reply:
/ngen/src/realizations/catchment/Bmi_Py_Formulation.cpp:50:92: error: no matching function for call to ‘models::bmi::Bmi_Py_Adapter::Bmi_Py_Adapter(std::string&, std::string&)’
   50 |         set_bmi_model(std::make_shared<Bmi_Py_Adapter>(Bmi_Py_Adapter(py_class, config_file)));